microk8s-resources/actions/enable.kubeflow.sh

```diff
@@ -108,6 +110,24 @@ def main():
     no_proxy = os.environ.get("KUBEFLOW_NO_PROXY") or None
     hostname = os.environ.get("KUBEFLOW_HOSTNAME") or None
     debug = strtobool(os.environ.get("KUBEFLOW_DEBUG") or "false")
+    ignore_min_mem = strtobool(os.environ.get("KUBEFLOW_IGNORE_MIN_MEM") or "false")
+
+    with open("/proc/meminfo") as f:
```

Comment:
@joedborg I wonder if we will get into trouble with strict confinement if we try to reacj `/proc/meminfo`. Do you know that?

Reply:
I guess we will care about this when the time comes.